certified-connectors/HrFlow/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1012 @@
+            "type": "integer",
+            "format": "int32",
+            "required": false,
+            "description": "To enable sync parsing you set this value to 1 otherwise the value is 0.",
+            "x-ms-summary": "sync_parsing.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "formData",
+            "name": "sync_parsing_indexing",
+            "type": "integer",
+            "format": "int32",
+            "required": false,
+            "description": "This parameter is used when you are using sync parsing it enables Profile indexing so this object will be available using search engine. The default value is 0",
+            "x-ms-summary": "sync_parsing_indexing.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "formData",
+            "name": "webhook_parsing_sending",
+            "type": "integer",
+            "format": "int32",
+            "required": false,
+            "description": "To enable the reception of webhook notification after parsing and before indexing you set the value to 1. The default value is 0.",
+            "x-ms-summary": "webhook_parsing_sending.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "formData",
+            "name": "labels",
+            "type": "string",
+            "required": false,
+            "description": "The Profile's labels to include with the object.",
+            "x-ms-summary": "labels.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "formData",
+            "name": "tags",
+            "type": "string",
+            "required": false,
+            "description": "The Profile's tags aims to mark uploaded object.",
+            "x-ms-summary": "tags.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "in": "formData",
+            "name": "metadatas",
+            "type": "string",
+            "required": false,
+            "description": "The Profile's metadata.",
+            "x-ms-summary": "metadatas.",
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Profile successfully Created using sync parsing.",
+            "schema": {
+              "$ref": "#/definitions/ProfileSyncParsingResponse"
+            }
+          },
+          "202": {
+            "description": "Profile successfully Created using async parsing.",
+            "schema": {
+              "$ref": "#/definitions/ProfileAsyncParsingResponse"
+            }
+          },
+          "401": {
+            "description": "Invalid secret key.",
+            "schema": {
+              "$ref": "#/definitions/Unauthorized"
+            }
+          }
+        }
+      }
+    },
+    "/v1/document/parsing": {
+      "post": {
+        "summary": "This operation returns document's parsing",
```

Comment:
@alaeddinechhima This summary value sounds like a description rather than a summary. Can you please improve it something like below:

`"summary": "Get Document's parsing"`